tests/unit/test_credentials.py

```diff
@@ -1,10 +1,13 @@
 import datetime
 from pathlib import Path
 from textwrap import dedent
+from typing import AsyncGenerator, Optional
 
 import pytest
+from _pytest.monkeypatch import MonkeyPatch
 from aiohttp import web
 from freezegun import freeze_time
+from pyfakefs.fake_filesystem import FakeFilesystem  # type: ignore
```

Comment:
I usually prefer using explicit ignores so specifying the failure kind: `# type: ignore[import]`. Same comment for other places with ignores here. But this is not too important to fix as it depends on the conventions of the codebase

Reply:
wasn't aware of that feature in mypy, thanks for sharing. updated all the ignore comments to ignore a specific error.